Arduino-Bling/RobotBling/animations.ino

```diff
@@ -0,0 +1,37 @@
+/*
+ * Robot animations
+ */
+
+/*
+ * Solid colors - remember to add 'delay()' to listen for new commands
+ */
+
+void solid(rgbColor c)
+{
+  setAllPixels(c.color);
+  showPixels();
+  delay(10);
```

Comment:
I think this 'delay(10);' should be replaced with:
```
// Wait forever, until a new command comes from the robot
while (1) {
  delay(10);
}
```

Reply:
That said, your method works fine. We could even add a check for commands at the end of the loop so that no delay is required, but then there is a lot of energy spent updating the display over and over...

Reply:
Will punt on this for now...